frontend/src/metabase/actions/components/ActionViz/ActionButton.styled.tsx

```diff
@@ -8,12 +8,14 @@ export const StyledButton = styled(Button)<{
   focus?: boolean;
 }>`
   height: ${({ isFullHeight }) => (isFullHeight ? "100%" : "auto")};
+
   ${({ focus }) =>
     focus
       ? `
     border: 2px solid ${color("focus")};
   `
       : ""}
+  padding: 0;
```

Comment:
nit: I'd put this above the above block, so we know which property applies to all conditions. So usually I'd see we structure styled component properties like this
1. plain properties
2. properties under selectors
3. block with `css` tag

and breakpoint or conditional styles somewhere below the 1)